CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## v0.24.0 [2022-02-09]
+
+_What's new?_
```

Comment:
I think I'd put this under our Deprecation section instead, since it's not really a new feature